### DISCUSSION TITLE
Is there something similar to radix-vue tags input in NuxtUI?

### DISCUSSION THREAD

**Discussion by CyberSekure** [nuxt/nuxt#26273] [Questions] [answered]
I'm looking for something similar to the radix-vue tags input (https://www.radix-vue.com/components/tags-input.html) in NuxtUI, but can't seem to spot anything. Any ideas?

Many thanks!

**Comment by jwhazel** [comment, marked as answer]
So there's nothing native _today_. It looks to me like the way to do it would be to use SelectMenu and combining these 3 things:
* Searchable to allow user to search: https://ui.nuxt.com/components/select-menu#searchable
* Multiple to allow user to select multiple tags: https://ui.nuxt.com/components/select-menu#multiple
* Override the label slot to render Badges https://ui.nuxt.com/components/select-menu#slots

Future state:
I suspect it could be added in the eventually.
* Feature request is here: https://volta.net/nuxt/ui/issues/931
* Radix Vue is already in the process of being used to replace some of the HeadlessUI components: https://volta.net/nuxt/ui/issues/409

**Reply by CyberSekure** [reply]
Many thanks for this. I'll take a look into what you suggested 👍

**Reply by akarelas** [reply]
@jwhazel It seems to me that creatable (which I think is most common in tag inputs) does not work with searchable & multiple. While the "create "foo"" row appears, neither hitting enter nor clicking on that row does anything. What's your opinion? Is this a bug?
